projects/js-packages/connection/hooks/use-product-checkout-workflow/index.jsx

```diff
@@ -13,7 +14,9 @@ const {
 	apiRoot,
 	apiNonce,
 	siteSuffix: defaultSiteSuffix,
-} = window?.JP_CONNECTION_INITIAL_STATE ? window.JP_CONNECTION_INITIAL_STATE : {};
+} = window?.JP_CONNECTION_INITIAL_STATE
```

Comment:
I'm wondering if we can extract this logic into a function. But I reckon it may be overkill if we're soon gonna discard `JP_CONNECTION_INITIAL_STATE`.

Reply:
Yeah, the plan is to ultimately get rid of `JP_CONNECTION_INITIAL_STATE`